src/app/stack/pipeline-insights/pipeline-insights.component.ts

```diff
@@ -13,35 +16,81 @@ import { PipelineInsightsService } from './pipeline-insights.service';
     styleUrls: ['./pipeline-insights.component.less'],
     templateUrl: './pipeline-insights.component.html'
 })
+@Injectable()
+// export class UserService {
+//   constructor (
+    
+//   ) {}
 
-export class PipelineInsightsComponent implements OnChanges {
+// }
+export class PipelineInsightsComponent implements OnInit, OnChanges {
     // @Input() gatewayConfig: any;
     @Input() stack: string;
     @Input() buildNumber;
     @Input() appName;
     @Input() stackResponse;
+    @Input() component;
+    @Input() url: string;
 
     @ViewChild('stackModule') modalStackModule: any;
 
-    public cve_Info: number;
-
+    public cve_Info: string;
+    public stackUrl: string;
+    public symbol: string = "pficon-error-circle-o";
 
+    constructor(private PipelineInsightsService: PipelineInsightsService,
+        private http: Http) {}
+    
+    
     public showStackModal(event: Event): void {
         event.preventDefault();
         this.modalStackModule.open();
     }
 
-    public getcve(data: number): void {
-        this.cve_Info = data;
+    public geturl():void {
+        this.stackUrl = this.url;
+        let observable: any = this  .PipelineInsightsService
+                                    .getStackAnalyses(this.url);
+
+        observable.subscribe((data) => {
+            if(data) {
+            console.log(data);
+            // let cve: number = data.result.data[0].user_stack_info.analyzed_dependencies.data[0].security;
+            let cve: string = data.result[0].user_stack_info.analyzed_dependencies[0].security;
```

Comment:
Put checks so that it doesn't break

Reply:
Please check for all results (manifests). The intention is to highlight even if there is one issues in any manifest.

Reply:
security will be array of objects(CVE, CVSS)